plugin/metrics/prometheus/metricsstore/reader.go

```diff
@@ -211,7 +211,38 @@ func (m MetricsReader) GetErrorRates(ctx context.Context, requestParams *metrics
 			)
 		},
 	}
-	return m.executeQuery(ctx, metricsParams)
+	errorMetrics, err := m.executeQuery(ctx, metricsParams)
```

Comment:
This is the main change, the rest supports this change, mostly for testing purposes.